packages/shared/src/utils/files/file-uploader-utils.ts

```diff
@@ -1,62 +1,39 @@
-import { compressImg, convertToBase64, isImageType, getFormatFromMIME, TImage, TFile } from './image/image_utility';
-
-export type TSettings = {
-    documentType: {
-        passport: string;
-        national_identity_card: string;
-        driving_licence: string;
-        utility_bill: string;
-        bankstatement: string;
-        power_of_attorney: string;
-        amlglobalcheck: string;
-        docverification: string;
-        proofid: string;
-        driverslicense: string;
-        proofaddress: string;
-        other: string;
-    };
-    pageType: {
-        front: string;
-        back: string;
-        photo: string;
-    };
-    expirationDate?: string;
-    documentId?: string;
-    lifetimeValid?: boolean;
-    document_issuing_country?: string;
-};
+import { useMutation } from '@deriv/api';
+import { compressImg, convertToBase64, isImageType, getFormatFromMIME, TImage } from './image/image_utility';
+
+export type TSettings = NonNullable<
+    NonNullable<NonNullable<Parameters<ReturnType<typeof useMutation<'document_upload'>>['mutate']>>[0]>['payload']
```

Comment:
here the same: we can use DeepRequired<>

Reply:
Thanks @yauheni-deriv